examples/integrations/gcp/src/jobs/etl/images/bounding_box/bounding_box_etl.py

```diff
@@ -61,7 +69,12 @@ def upload_to_gcs(image_bytes: BytesIO, data_row_uid: str, h: int, w: int,
     return f"gs://{bucket.name}/{blob.name}"
 
 
-def process_label(label: Label, bucket: Bucket) -> str:
+def process_label(label: Label, partition: Optional[Partition],
+                  bucket: Bucket) -> str:
+    if partition is None:
+        logger.warning("No partition assigned. Skipping.")
+        return
+
```

Comment:
I figured something like this was missing previously.
But this will never be the case, right? There's always going to be a split, even if a default one, correct?

Reply:
Unfortunately it can be empty.. If they launch the job from the UI it won't be through